packages/gatsby/src/utils/api-browser-docs.js

```diff
@@ -1,5 +1,6 @@
 /**
  * Called when the Gatsby browser runtime first starts.
+ * @param {Object} pluginOptions
```

Comment:
This is weird, but `pluginOptions` I think will be second paramater, and first one will actually be undefined. And this is the same for all APis that doesn't really pass any args (so to `onInitialClientRender`, `registerServiceWorker`, `disableCorePrefetching`, `replaceHydrateFunction` as well)

Reply:
Agreed a little weird, but correct.  I pulled down your update and added 

`@param undefined _ `

for the first parameter on those you listed (not sure that is the best way to display that though), and including the `@param {Object} pluginOptions after.  I then tried `gatsby develop` just to be sure everything was good, and I'm getting the error: 

```
GraphQL Error Fragment "DocumentationParamsFieldsFragment" cannot be spread here as objects of type "params_2" can never be of type "DocumentationJs".

  file: /Users/sbardian/stuff/dev/javascript/PRs/gatsby/www/src/components/api-reference/params.js

  35 |           static {
  36 |             ...DocumentationParamBase
  37 |           }
  38 |         }
  39 |       }
  40 |     }
  41 |   }
  42 |   fragment DocumentationParamsFragmentR1 on DocumentationJs {
  43 |     ...DocumentationParamsFieldsFragment
  44 |     params {
> 45 |       ...DocumentationParamsFieldsFragment
     |       ^
  46 |     }
  47 |     properties {
  48 |       ...DocumentationParamsFieldsFragment
  49 |     }
  50 |     members {
  51 |       static {
  52 |         ...DocumentationParamsFieldsFragment
  53 |       }
  54 |     }
  55 |     type {
```

Reply:
Seems to be something upstream.  Pushed and will rebase after it's fixed.